src/app/cluster/details/cluster/add-machine-network/component.spec.ts

```diff
@@ -35,19 +35,21 @@ describe('AddMachineNetworkComponent', () => {
   let component: AddMachineNetworkComponent;
   let fixture: ComponentFixture<AddMachineNetworkComponent>;
 
-  beforeEach(waitForAsync(() => {
-    TestBed.configureTestingModule({
-      imports: [...modules],
-      declarations: [AddMachineNetworkComponent, MachineNetworksComponent],
-      providers: [
-        WizardService,
-        {provide: ClusterService, useClass: ClusterMockService},
-        {provide: MatDialogRef, useClass: MatDialogRefMock},
-        NotificationService,
-      ],
-      teardown: {destroyAfterEach: false},
-    }).compileComponents();
-  }));
+  beforeEach(
```

Comment:
It looks that you have some different version of dependencies installed. Run `npm ci` each time you download latest changes from master. To check it now you can run `npm ci && npm run fix`.